Workaround for:
Exception in thread "main" java.lang.NoSuchFieldError: preferIPv6Address
        at java.base@9.0.4-internal/java.net.InetAddress.init(Native Method)
        at java.base@9.0.4-internal/java.net.InetAddress.<clinit>(InetAddress.java:336)
        at build.tools.classlist.HelloClasslist.main(HelloClasslist.java:60)

--- a/jdk/make/src/classes/build/tools/classlist/HelloClasslist.java    2021-12-07 19:18:50.615675432 +0000
+++ b/jdk/make/src/classes/build/tools/classlist/HelloClasslist.java    2021-12-07 19:19:28.706564894 +0000
@@ -56,8 +56,7 @@
 
     public static void main(String ... args) {
 
-        List<String> strings = Arrays.asList("Hello", "World!", "From: ",
-              InetAddress.getLoopbackAddress().toString());
+        List<String> strings = Arrays.asList("Hello", "World!");
 
         String helloWorld = strings.parallelStream()
               .map(s -> s.toLowerCase(Locale.ROOT))
